proxy-node-chart/charts/metadata-controller/templates/customer-ca-crt-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "metadata-controller.fullname" . }}
+  labels:
+    app: {{ template "metadata-controller.name" . }}
+    chart: {{ template "metadata-controller.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  hsm_signing_cert.pem: {{ required "A valid .Values.customerCaCrt entry required!" .Values.customerCaCrt | b64enc | quote }}
```

Comment:
You probably don't mean to use `customerCaCrt` here, right? The `hsm_signing_cert` is the cert attached to the private key in the HSM while the `customerCACrt` is used by the cloudhsm_client to access the HSM cluster.

Reply:
I ported the change to the incorrect configmap name + key